torch/_inductor/codegen/cpp.py

```diff
@@ -231,6 +231,30 @@ def erf(x):
     def sqrt(x):
         return f"{x}.sqrt()"
 
+    @staticmethod
+    def eq(x, y):
+        return f"{x} == {y}"
+
+    @staticmethod
+    def ne(x, y):
+        return f"{x} != {y}"
+
+    @staticmethod
+    def lt(x, y):
+        return f"{x} < {y}"
+
+    @staticmethod
+    def gt(x, y):
+        return f"{x} > {y}"
+
+    @staticmethod
+    def le(x, y):
+        return f"{x} <= {y}"
+
+    @staticmethod
+    def ge(x, y):
+        return f"{x} >= {y}"
+
```

Comment:
Have you verified TorchBench, HF and TIMM? Meanwhile, I would prefer conservative optimization from the functionality perspective. Could you please share which model requires this optimization?

Reply:
`aten.elu.default` needs `gt` and `aten.elu_backward.default` needs `le` for vectorization. The models containing these ops are `nvidia_deeprecommender` in TorchBench and `dpn107` in TIMM.

Reply:
This PR has been verified for all models of TorchBench, HF and TIMM.

Reply:
Just to be clear, what does "verified" mean here? Do you mean performance or correctness?

Reply:
"Verified" means correctness here.